src/Symfony/Component/String/Tests/AbstractUnicodeTestCase.php

```diff
@@ -405,6 +405,16 @@ public static function provideReplace(): array
         );
     }
 
+    public static function provideReplaceMatches(): array
+    {
+        return array_merge(
+            parent::provideReplaceMatches(),
+            [
+                ['This is a dj-vu situation!', 'This is a déjà-vu situation!', '/([à-ú])/', ''],
```

Comment:
Nice 😋😁

Reply:
what is the issue with continuous-integration/appveyor?